Libraries/MLXLMCommon/KVCache.swift

```diff
@@ -145,7 +145,7 @@ public func createCausalMask(n: Int, offset: Int, windowSize: Int? = nil) -> MLX
     var mask = linds .>= rinds
 
     if let windowSize {
-        mask = mask & (linds .< rinds + windowSize)
+        mask = mask & (linds .<= rinds + windowSize)
```

Comment:
Same as in python
https://github.com/ml-explore/mlx-lm/blob/f1a6359c92827f0ba2185f5264b9ddb313f7360e/mlx_lm/models/base.py#L38